### PR TITLE
Mention Open JDK in step by step exporting

### PR DESCRIPTION
**Pull request by skyace65**
Replace Oracle JDK with OpenJDK in step by step exporting. And clarifies that you must have version 8 on the android custom builds page.

### CODE REVIEW

File: getting_started/step_by_step/exporting.rst [hunk 1]
```diff
@@ -315,11 +315,12 @@ Android
...
 
-Before you can export your project for Android, you must download the following
-software:
+Before you can export your project for Android, you must download the
+Android SDK and a JDK (Oracle JDK or Open JDK).
 
 * Android SDK: https://developer.android.com/studio/
-* Java JDK: http://www.oracle.com/technetwork/java/javase/downloads/index.html
+* Oracle JDK: http://www.oracle.com/technetwork/java/javase/downloads/index.html
+* Open JDK(version 8 is recommended): https://adoptopenjdk.net/index.html
```
Comment (Calinou) [inline getting_started/step_by_step/exporting.rst:323]:
```suggestion
* or OpenJDK (version 8 is required, more recent versions won't work): https://adoptopenjdk.net/index.html
```

**Comment by skyace65** [comment]
@Calinou it now says version 8 is required in the exporting page, and I've changed it on the custom builds for android page to say that as well. Also I already mentioned either JDK would work above the links.

**Comment by akien-mga** [comment]
I would outright remove the reference to Oracle.

I'm not sure you can still get JDK 8 from them without paying.

**Comment by skyace65** [comment]
Ok, I've removed the references to the Oracle JDK.

**Pull request merged**